### PR TITLE
translate to norwegian

### PR DESCRIPTION
**Pull request by magdalenatran**

### CODE REVIEW

File: src/pages/regionPage.tsx [hunk 1]
```diff
@@ -15,12 +15,12 @@ const RegionPage = () => {
     const reduxState = useSelector((state: combinedStateInterface) => state);
     const dispatch = useDispatch();
 
-    let Nord: regionInterface = { id: 0, name: NORDNORGE, cities: [] };
-    let Midt: regionInterface = { id: 1, name: MIDTNORGE, cities: [] };
-    let Vest: regionInterface = { id: 2, name: VESTLANDET, cities: [] };
-    let Sor: regionInterface = { id: 3, name: SORLANDET, cities: [] };
-    let Ost: regionInterface = { id: 4, name: OSTLANDET, cities: [] };
-    let regions: regionInterface[] = [Nord, Midt, Vest, Sor, Ost];
+    const Nord: regionInterface = { id: 0, name: NORDNORGE, cities: [] };
```
Comment (KristofferHaakonsen) [inline src/pages/regionPage.tsx:18]: Nice

**Review by KristofferHaakonsen — APPROVED** [review]
Ser bra ut :)

**Review by KristofferHaakonsen — CHANGES_REQUESTED** [review]
Må håndtere merge conflict, men ellers bra :)

**Review by edvardag — APPROVED** [review]

**Review by KristofferHaakonsen — CHANGES_REQUESTED** [review]
Må merge sportPage.tsx

**Review by KristofferHaakonsen — APPROVED** [review]
Nice

**Pull request merged**